tark/translation/views.py

```diff
@@ -37,3 +37,18 @@ def get_queryset(self):
 class TranslationDetail(generics.RetrieveAPIView):
     queryset = Translation.objects.all()
     serializer_class = TranslationSerializer
+
+
+class TranslationListByStableID(generics.ListAPIView):
+    serializer_class = TranslationDetailSerializer
+
+    def get_queryset(self):
+        stable_id = self.kwargs.get('stable_id')
+        if not stable_id:
+            raise NotFound('No stable ID provided.')
+        
+        queryset = Translation.objects.filter(stable_id=stable_id).prefetch_related('translation_release_set').order_by('pk')
+        if queryset.exists():
+            return queryset
+        else:
+            raise NotFound('No translations found for the given stable ID.')
```

Comment:
The API will break when we don't provide a stable_id or when it doesn't exist
```
http://localhost:9000/api/translation/stable_id/dummy_stable_id/
```

> NameError at /api/translation/stable_id/dummy_stable_id/
> name 'NotFound' is not defined

---
You forgot to import `NotFound` Class
```
from rest_framework.exceptions import NotFound
```
This should fix it

Reply:
Thanks for catching it. I have updated the code.